src/main/java/io/pinecone/configs/PineconeConfig.java

```diff
@@ -207,6 +208,24 @@ public String getUserAgent() {
         return buildUserAgent();
     }
 
+    /**
+     * Returns true if TLS is enabled or false otherwise.
+     *
+     * @return enableTls
+     */
+    public boolean getEnableTLS() {
```

Comment:
just a note on the naming. Should probably be something like "tlsEnabled"

Reply:
Yes, tlsEnabled makes more sense but since the rest of the properties in this class have getters and setters with getProp1(), getProp2(), etc., I went with getEnableTLS(). But if you still feel that we should change the name to tlsEnable(), I can make that change.

Reply:
Most java naming guidelines would actually use `is` for the getter i.e. `isTLSEnabled`, and the `set` companion would be `setTLSEnabled`. I don't think it has to match with the other getters -- it's more important that it semantically makes sense.

If it was returning a nullable `Boolean` then getTLSEnabled would be appropriate